src/client/utils.js

```diff
@@ -0,0 +1,20 @@
+const fetchConfig = (reqBody, method) => {
+    return {
+        method: method,
+        headers: {
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(reqBody),
+    };
+};
+
+export const doFetch = async (endpoint, reqBody, method) => {
+    try {
+        const response = await fetch(endpoint, fetchConfig(reqBody, method));
+        return await response.json();
+    } catch (error) {
+        console.log(error);
+    }
+};
+
+export const capitaliseFirstLetter = (string) => string.replace(/\b\w/g, (c) => c.toUpperCase());
```

Comment:
This looks like it belongs in a different file - turn this file into a directory and create more cleanly separated utility functions by purpose